### PR TITLE
Plot Cumulative Return for each period

### PR DESCRIPTION
**Pull request by luca-s**
Cumulative Return by Quantile plot for each period and
Factor Weighted Cumulative Return plot for each period

Issue #66 and #65

### CODE REVIEW

**Comment by luca-s** [comment]
untitled

File: alphalens/plotting.py [hunk 1]
```diff
@@ -709,7 +721,15 @@ def plot_cumulative_returns_by_quantile(quantile_returns, ax=None):
...
 
     ret_wide = quantile_returns.reset_index()\
-        .pivot(index='date', columns='quantile', values=1)
+        .pivot(index='date', columns='quantile', values=period)
+
+    if period > 1:
+        daily_returns = lambda ret, period: (
+            np.nanmean(ret) ** (1. / period)) - 1
+        period_ret_wide = ret_wide.add(1)**period
+        ret_wide = period_ret_wide.rolling(
```
Comment (jameschristopher) [inline alphalens/plotting.py]: Is this way of doing rolling operations pandas 16 compatible? I thought this was just recently introduced.

**Comment by jameschristopher** [comment]
This is awesome

File: alphalens/plotting.py (hunk 1 above)
Comment (luca-s) [inline alphalens/plotting.py]: Yeah, it's a pandas 18 (18.1?) feature. I wasn't sure what was the target pandas version. I will use the old method then (and eventually we have to switch back to this version as the new one complains about deprecated pandas.rolling_apply :) )

Comment (luca-s) [inline alphalens/plotting.py]: it should be fine now

**Comment by luca-s** [comment]
That makes sense, I agree with you. I will do that then.

**Comment by jameschristopher** [comment]
LGTM

**Comment by twiecki** [comment]
:+1:

**Pull request merged**